app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -3420,6 +3421,38 @@ class PluginApi {
     registerRichEditorExtension(extension);
   }
 
+  /**
+   * Adds a card to the admin themes grid.
+   *
+   * ```js
+   * import Component from "@glimmer/component";
+   *
+   * api.addCardToAdminThemesGrid((adminConfigAreaCardComponent) => {
+   *   return class extends Component {
+   *     <template>
+   *       <adminConfigAreaCardComponent class="my-custom-card">
+   *         <:content>
+   *           <!-- your custom card content goes here -->
+   *         </:content>
+   *       </adminConfigAreaCardComponent>
+   *     </template>
+   *   }
+   * });
+   * ```
+   *
+   * @param {Function} func - A function that returns a card component class.
+   *
+   * @callback func
+   * @param {AdminConfigAreaCard} adminConfigAreaCardComponent - The
+   * `AdminConfigAreaCard` class from core to be optionally used as a container
+   * for the card content.
+   * @returns {Component} - A Glimmer component class for the card to be added
```

Comment:
I don't think we return a `Component` here but a function. I think we can just leave the return value blank here since nothing we return to the function caller is of use.